SAPHana/ra/SAPHanaController

```diff
@@ -1123,7 +1178,9 @@ function register_hana_secondary()
                remoteHanaHost=$remoteHost
            fi
            super_ocf_log info "ACT: SAPHANA REGISTER: hdbnsutil -sr_register --remoteHost=$remoteHanaHost --remoteInstance=$remoteInstance $hanaRM $hanaOM --name=$sr_name"
+           set_hana_attribute "$NODENAME" "R" "${ATTR_NAME_HANA_SRACTION[@]}"
            HANA_CALL --timeout inf --use-su --cmd "hdbnsutil -sr_register --remoteHost=$remoteHanaHost --remoteInstance=$remoteInstance $hanaRM $hanaOM --name=$sr_name"; rc=$?
+           set_hana_attribute "$NODENAME" "-" "${ATTR_NAME_HANA_SRACTION[@]}"
```

Comment:
Wenn sr_register einen Fehler liefert (rc !=0), sollte dann das Attribut auf '-' gesetzt werden oder könnte man  dann hier ein anderes Attribut setzten, um diesen Fehler auch in den Attributen zu reflektieren?

Reply:
Gute Idee. Wir müssten dann nochmal darüber nachdenken, wer einen solchen "Fehler-Status" dann auch wieder zurücksetzt. Beim Scheitern des sr_register wie auch des sr_takeover macht der Cluster eh hässliche Dinge. Die Resource wird meines Wissens abgeräumt.